src/Providers/ApiProvider.js

```diff
@@ -0,0 +1,32 @@
+import { createContext, useContext, useEffect, useState } from "react"
```

Comment:
I've already given you some feedback regarding the providers in Slack, the same thing applies here :)